Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml

```diff
@@ -76,6 +75,14 @@ configuration:
   type: 8
   section: Connect
   advanced: true
+- display: Incident type
```

Comment:
why move this?

Reply:
validation error for the previous Incident type (IN124 I think)